lib/inject.js

```diff
@@ -26,7 +26,7 @@ function injectPa11y(window, options, done) {
 	}
 
 	function processMessages(messages) {
-		return messages.map(processMessage).filter(isMessageWanted);
+		return messages.filter(isMessageInTestArea).map(processMessage).filter(isMessageWanted);
```

Comment:
Oh actually one more small tweak. Currently if there is no `rootElement` specified, we run `querySelector` on `null`. I wonder whether this function would be more optimal if it was:

```js
function processMessages(messages) {
	if (options.rootElement) {
		messages = messages.filter(isMessageInTestArea);
	}
	return messages.map(processMessage).filter(isMessageWanted);
}
```